### DISCUSSION TITLE
In Terminal copy and paste

### DISCUSSION THREAD

**Discussion by danny1209** [OE4T/tegra-demo-distro#236] [Q&A] [answered]
Hi, 
I compiled demo-image-full.
my platform is conntected with a display, I see in the Terminal is not possible to copy/paste text. Everything must be typed manually.
But in JetPack it´s possible to copy/paste. 

Is there any solution to make copy/paste working in the Terminal?

**Comment by madisongh** [comment, marked as answer]
It's like xterm... select (using left mouse button) puts the text into a 'cut buffer', which you can paste by pressing the middle mouse button (or, IIRC, shift-Insert, if you don't have a middle button).  Google for 'xterm copy paste' for other tips.

**Reply by danny1209** [reply]
Nice! it works.
